CMakeLists.txt

```diff
@@ -0,0 +1,234 @@
+cmake_minimum_required (VERSION 2.8.11)
+project (libpd C)
+
+option(BUILD_SHARED_LIBS "Build libpd as a shared library instead of static" ON)
+option(pd_utils "Compile utilities" ON)
+option(pd_extra "Compile extras" ON)
+option(pd_multi "Compile with multiple instance support" ON)
```

Comment:
I think it should be off by default

Reply:
Yes. It should be an option but not default behavior.

Reply:
sure